R/loo.R

```diff
@@ -215,16 +218,18 @@ loo.matrix <-
            ...,
            r_eff = NULL,
            save_psis = FALSE,
-           cores = getOption("mc.cores", 1)) {
-
+           cores = getOption("mc.cores", 1),
+           is_method = c("psis", "tis", "sis")) {
```

Comment:
So it has been changed to method in `importance_sampling()` constructor since there `method` makes sense (it is the importance sampling method used). But in loo this becomes more complicated.  Using method is semantically slightly difference since there it would be the loo method. To me that could be a naive approach, approximation using Taylor series (as in waic)    etc. So in loo I kept `is_method` to make it more explicit. Maybe call it `importance_sampling_method` but that may be too long?

Reply:
Ah I see. Ok, from this perspective `is_method` is fine for me.